web/src/views/admin/Admin.vue

```diff
@@ -1,24 +0,0 @@
-<template>
```

Comment:
Is the change to the admin view and router somehow related to the restructured user settings, or just a "drive-by" cleanup?